fvm/evm/stdlib/contract.go

```diff
@@ -97,6 +99,48 @@ func (e abiDecodingError) Error() string {
 	return b.String()
 }
 
+func calculateComputation(
+	inter *interpreter.Interpreter,
+	values *interpreter.ArrayValue,
+	evmAddressTypeID common.TypeID,
+) int {
```

Comment:
wouldn't it be better to return type of `uint`?

Reply:
Good point :+1:

Reply:
Updated in https://github.com/onflow/flow-go/pull/5146/commits/5a6dfa5545d0dfebd4258dbebe9319c930c4d5bf